### PR TITLE
fix(PDEV-1370): Updated API docs as per the new changes in the v4 API for External Funding Account. Added GET method

### PR DESCRIPTION
**Pull request by prachisshah**

### CODE REVIEW

File: api/v10/index.raml [hunk 1]
```diff
@@ -702,11 +702,17 @@
                 application/json:
                   example: !include examples/getExternalFundingAccounts.res.json
                   schema: !include schema/getExternalFundingAccounts.res.json
+            204:
```
Comment (yuan616510) [inline api/v10/index.raml]: Should be vertically aligned with 200 instead of "body"

Comment (yuan616510) [reply api/v10/index.raml]: Need to fix the same issue for the rest of parts.

Comment (yuan616510) [reply api/v10/index.raml]: Ignore the last two comments. My browser issue.

Comment (prachisshah) [reply api/v10/index.raml]: The code is aligned. Please view the file to see the aligned changes.

**Comment by prachisshah** [comment]
Merging, closing PR, deleting branch.

**Pull request merged**